proto/types/blockchain.proto

```diff
@@ -63,12 +63,19 @@ message CandidateList {
   repeated Candidate candidates = 1;
 }
 
+message EpochData {
+  uint64 num = 1;
+  uint64 height = 2;
```

Comment:
Is it the start height of each block, or current block height?

Reply:
it is the start height, same as we defined in epochCtx